testUtils/handlerInputesBuilder.js

```diff
@@ -4,13 +4,15 @@ export class HandlerInputesBuilder {
   static DEFAULT_AWS_REQUEST_ID = HttpSpanBuilder.DEFAULT_PARENT_ID;
   static DEFAULT_INVOKED_FUNCTION_ARN = HttpSpanBuilder.DEFAULT_ARN;
   static DEFAULT_FUNCTION_VERSION = HttpSpanBuilder.DEFAULT_VERSION;
+  static DEFAULT_TIMEOUT = 10 * 1000;
```

Comment:
This is only for tests. Right?
If we can't find the timeout for the client we're not putting 10 seconds timer?

Reply:
Yes